.github/workflows/update-wasm-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Update Wasm tests
+
+on:
+  # Trigger at every Sunday UTC noon, or manually.
+  schedule:
+    - cron: 0 12 * * 0
+  workflow_dispatch:
+
+jobs:
+  build-wpt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout WPT repo
+        uses: actions/checkout@v4
+        with:
+          path: main
+      - name: Checkout Wasm repo
+        uses: actions/checkout@v4
+        with:
+          repository: WebAssembly/spec
+          path: wasm-spec
+      - name: Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14.x
+      - name: Setup OCaml tools
+        run: opam install --yes ocamlfind.1.9.5 js_of_ocaml.4.0.0 js_of_ocaml-ppx.4.0.0
+      - name: Build interpreter
+        run: cd wasm-spec/interpreter && opam exec make
+      - name: Convert WAST tests to WPT
+        run: wasm-spec/test/build.py --dont-recompile --html wasm-spec/out/
+      - name: Copy Wasm tests to WPT
+        # Replace wasm/core entirely and fix up testharness paths.
+        run: |
+          rm -rf main/wasm/core
+          cp -r wasm-spec/out/ main/wasm/core/
+          find main/wasm/core/ -type f -name '*.html' -exec sed -i 's/\.\/js\/harness\/testharness/\/resources\/testharness/' {} \;
```

Comment:
I mean, OK, but this feels like a footgun. Ideally the build step would do this for us, rather than doing string substitution on the built files. If we must do this then we could at least use `'s|input|output|'` or something to avoid needing all the escaping of slashes.

Reply:
I think we can probably fix this in the build script. Let me take a look.

Reply:
I landed https://github.com/WebAssembly/spec/pull/1859 which should match what you have here. I will make further refinements in followup PRs (e.g. there are some harness files that we probably don't need to copy).

Reply:
Thanks for changing build.py Derek! Removing this step is a nice simplification. Tested it in my own fork and it worked flawlessly.